src/bosquet/template/tag.clj

```diff
@@ -12,5 +12,8 @@
 
 (defn add-tags []
   (parser/add-tag! :llm-generate
-    (fn [args {pretext :selmer/preceding-text}]
-      (openai/complete pretext (args->map args)))))
+    (fn [args context]
```

Comment:
here I merge the "model opts" passed in by the user and the model opts from the prompt definition